git-scripts/git-vgrep

```diff
@@ -3,8 +3,22 @@
 set -e
 out=$(mktemp -t git-vgrep-XXXXXX)
 trap "rm -f $out" EXIT
-git grep -n "$@" | tee "$out"
-if [ "$(cat "$out" | wc -l)" -eq 1 ] ; then
+git grep -n "$@" | tee "$out" 1> /dev/null
+
+lines=$(cat -b "$out" | wc -l)
+
+if [ $lines -eq 0 ]; then
+	exit 0
```

Comment:
Honestly, it seems like this should be `exit 1` since we failed to find anything? Probably doesn't matter...

Reply:
I changed to 1.